### PR TITLE
feat: [#175898227] implements delete button for credit card

### PR DESCRIPTION
**Pull request by qesya**
## Short description
implements delete button for the credit card
Screenshot 2021-02-04 at 23 31 10
Screenshot 2021-02-04 at 23 31 19

### CODE REVIEW

File: ts/screens/wallet/TransactionsScreen.tsx [hunk 1]
```diff
@@ -111,79 +137,118 @@ class TransactionsScreen extends React.Component<Props> {
...
+  potPayments: paymentsHistorySelector(state),
+  transactionsLoadedLength: getTransactionsLoadedLength(state),
   favoriteWallet: getFavoriteWalletId(state),
+  readTransactions: state.entities.transactionsRead,
+  areMoreTransactionsAvailable: areMoreTransactionsAvailable(state),
   paymentMethods: paymentMethodsSelector(state)
 });
 
 const mapDispatchToProps = (dispatch: Dispatch) => ({
+  loadTransactions: (start: number) =>
```
Comment (debiff) [inline ts/screens/wallet/TransactionsScreen.tsx:232]: Why you reintroduced this logic? You removed in #2628 
```suggestion
```

File: ts/screens/wallet/TransactionsScreen.tsx [hunk 2]
```diff
@@ -111,79 +137,118 @@ class TransactionsScreen extends React.Component<Props> {
...
+    dispatch(fetchTransactionsRequest({ start })),
+  navigateToTransactionDetailsScreen: (transaction: Transaction) => {
```
Comment (debiff) [inline ts/screens/wallet/TransactionsScreen.tsx:234]: Why you reintroduced this logic? You removed in #2628 
```suggestion
```

File: ts/screens/wallet/TransactionsScreen.tsx [hunk 3]
```diff
@@ -111,79 +137,118 @@ class TransactionsScreen extends React.Component<Props> {
...
     dispatch(setFavouriteWalletRequest(walletId)),
   deleteWallet: (walletId: number) =>
     dispatch(
       deleteWalletRequest({
         walletId,
-        onSuccess: action => {
+        onSuccess: _ => {
           showToast(I18n.t("wallet.delete.successful"), "success");
-          if (action.payload.length > 0) {
-            dispatch(navigateToWalletList());
```
Comment (debiff) [inline ts/screens/wallet/TransactionsScreen.tsx:183]: Why you removed this behaviour? Is expected by the design team?

**Review by debiff — CHANGES_REQUESTED** [review]

File: ts/screens/wallet/TransactionsScreen.tsx (hunk 3 above)
Comment (qesya) [reply ts/screens/wallet/TransactionsScreen.tsx:183]: action.payload's always not empty

**Comment by debiff** [comment]
Solved in #2917

**Pull request closed without merging**